test/powershell/Language/Scripting/Requires.Tests.ps1

```diff
@@ -0,0 +1,28 @@
+Describe "Requires tests" -Tags "CI" {
+    Context "Parser error" {
+        
+        $testcases = @(
+                        @{command = "#requiresappID`r`n$foo = 1; $foo" ; testname = "appId with newline"}
```

Comment:
You use double quoted strings, but `$foo` is not defined, so you're not parsing what you think you are.